.github/workflows/main.yml

```diff
@@ -0,0 +1,123 @@
+          restore-keys: |
+            ${{ runner.os }}-install-${{ env.cache-name }}-
+            ${{ runner.os }}-install-
+            ${{ runner.os }}-
+      - name: Install package
+        run: |
+          python -m pip install --upgrade pip
+          pip install --user picamera
+          pip install pyronear
+
+  pkg-test:
+    runs-on: ubuntu-latest
+    needs: pkg-install
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+          architecture: x64
+      - name: Cache python modules
+        uses: actions/cache@v1.1.2
+        env:
+          cache-name: cache-pkg-requirements
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-install-${{ env.cache-name }}-${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-install-${{ env.cache-name }}-
+            ${{ runner.os }}-install-
+            ${{ runner.os }}-
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pyronear
+          pip install --user picamera
+          pip install -r .github/workflows/requirements.txt
+
+      - name: Run unittests
+        run: |
+          coverage run -m unittest discover test/
+          coverage xml
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1.0.6
+        with:
+          file: ./coverage.xml
+          flags: unittests
+          fail_ci_if_error: true
+
+  flake8-py3:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+          architecture: x64
+      - name: Run flake8
+        run: |
+          pip install -r .github/workflows/requirements.txt
+          flake8 --version
+          flake8 ./
+
+  docs-build:
+    runs-on: ubuntu-latest
+    needs: pkg-install
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+          architecture: x64
+      - name: Cache python modules
+        uses: actions/cache@v1.1.2
+        env:
+          cache-name: cache-pkg-requirements
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-install-${{ env.cache-name }}-${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-install-${{ env.cache-name }}-
+            ${{ runner.os }}-install-
+            ${{ runner.os }}-
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pyronear
+          pip install --user picamera
+          pip install -r docs/requirements.txt
+
+      - name: Build documentation
+        run: |
+          sphinx-build docs/source docs/build -a
```

Comment:
You can remove this for now since documentation cannot be built

Reply:
Done